sdk/formrecognizer/azure-ai-formrecognizer/tests/test_mgmt.py

```diff
@@ -0,0 +1,101 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+import functools
+from azure.core.credentials import AzureKeyCredential
+from azure.core.exceptions import ResourceNotFoundError
+from azure.core.pipeline.transport import RequestsTransport
+from azure.ai.formrecognizer import FormTrainingClient, FormRecognizerClient
+from testcase import FormRecognizerTest, GlobalFormRecognizerAccountPreparer, GlobalFormAndStorageAccountPreparer
+from testcase import GlobalTrainingAccountPreparer as _GlobalTrainingAccountPreparer
+
+
+GlobalTrainingAccountPreparer = functools.partial(_GlobalTrainingAccountPreparer, FormTrainingClient)
+
+
+class TestManagement(FormRecognizerTest):
+
+    @GlobalFormRecognizerAccountPreparer()
+    def test_account_properties(self, resource_group, location, form_recognizer_account, form_recognizer_account_key):
+        client = FormTrainingClient(form_recognizer_account, AzureKeyCredential(form_recognizer_account_key))
+        properties = client.get_account_properties()
+
+        self.assertIsNotNone(properties.custom_model_limit)
+        self.assertIsNotNone(properties.custom_model_count)
+
+    @GlobalFormAndStorageAccountPreparer()
+    @GlobalTrainingAccountPreparer()
+    def test_mgmt_model(self, client, container_sas_url):
```

Comment:
I think you should split this up, so one for labeled, one for unlabeled

Reply:
you noticed my laziest. :) Done.